src/plugins/addimages.js

```diff
@@ -7,6 +7,15 @@
         return;
     }
 
+    else {
+        window.addEventListener("dragover",function(event){
```

Comment:
Could you please reformat this?

- Use single quotes for strings
- Add spaces after comas `, `
- Add a space before the opening braces/curly brackets of functions? 

I think that if you look at the rest of the code, it should be easy to figure out, but if you have a doubt you can always ask me for help.

Thanks!